content/en/docs/concepts/security/service-accounts.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Service Accounts
+description: >
+  Learn about the Kubernetes ServiceAccount object.
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+This page introduces the ServiceAccount object in Kubernetes, providing
+information about how service accounts work, use cases, limitations,
+alternatives, and links to resources for additional guidance.
+
+<!-- body -->
+
+## What are service accounts?
+
+A service account is a type of non-human account that, in Kubernetes, provides
+an identity for the processes that run in Pods. Pods can use service accounts
+to make calls to the Kubernetes API server. Service accounts exist as
+ServiceAccount objects in the API server. Service accounts have the following
+properties:
+
+*  **Namespaced:** Each service account is bound to a Kubernetes
+   {{<glossary_tooltip text="namespace" term_id="namespace">}}. Every namespace
+   gets a `default` ServiceAccount upon creation.
```

Comment:
is it clear that `default` is the actual name of the service account?

Reply:
Rephrased